snforge_std/src/cheatcodes.cairo

```diff
@@ -19,6 +19,12 @@ struct ContractClass {
     class_hash: ClassHash,
 }
 
+#[derive(Drop, Clone)]
+struct L1HandlerFn {
+    contract_address: ContractAddress,
+    selector_name: felt252,
```

Comment:
Is `selector_name` ok, or `function_name` would be better?

Reply:
`function_name` is more accurate, selector is the value of hash